[2025-07-30T11:40:30.321+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-30T11:40:30.370+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: financial_etl_pipeline.load_data_to_postgres manual__2025-07-30T11:40:21.134064+00:00 [queued]>
[2025-07-30T11:40:30.384+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: financial_etl_pipeline.load_data_to_postgres manual__2025-07-30T11:40:21.134064+00:00 [queued]>
[2025-07-30T11:40:30.385+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2025-07-30T11:40:30.585+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_data_to_postgres> on 2025-07-30 11:40:21.134064+00:00
[2025-07-30T11:40:30.605+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'financial_etl_pipeline', 'load_data_to_postgres', 'manual__2025-07-30T11:40:21.134064+00:00', '--job-id', '243', '--raw', '--subdir', 'DAGS_FOLDER/financial_etl_dag.py', '--cfg-path', '/tmp/tmpxbtazcko']
[2025-07-30T11:40:30.611+0000] {standard_task_runner.py:91} INFO - Job 243: Subtask load_data_to_postgres
[2025-07-30T11:40:30.612+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1817) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-30T11:40:30.614+0000] {standard_task_runner.py:63} INFO - Started process 1829 to run task
[2025-07-30T11:40:30.725+0000] {task_command.py:426} INFO - Running <TaskInstance: financial_etl_pipeline.load_data_to_postgres manual__2025-07-30T11:40:21.134064+00:00 [running]> on host 9c353dd7cf4b
[2025-07-30T11:40:30.834+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='financial_etl_pipeline' AIRFLOW_CTX_TASK_ID='load_data_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-07-30T11:40:21.134064+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-30T11:40:21.134064+00:00'
[2025-07-30T11:40:30.835+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-30T11:40:30.894+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-30T11:40:30.896+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-30T11:40:30.914+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=financial_etl_pipeline, task_id=load_data_to_postgres, execution_date=20250730T114021, start_date=20250730T114030, end_date=20250730T114030
[2025-07-30T11:40:30.950+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-30T11:40:30.974+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-30T11:40:30.977+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
